daemons/gptp/linux/src/daemon_cl.cpp

```diff
@@ -99,6 +99,8 @@ static IEEE1588Port *pPort = NULL;
 
 int main(int argc, char **argv)
 {
+	GPTP_LOG_INFO("gPTP starting");
```

Comment:
For some reason, this file is indented by spaces rather than tabs, so this line does not match the formatting of the rest of the file. Probably best to keep file-level consistency and possibly switch this file back to consistency with the rest of the gptp code another time.